README.md

```diff
@@ -10,7 +10,7 @@ The Movement SDK is a collection of tools and libraries for building, deploying,
 - [`util`](./util): Utility crates for the Movement SDK. These crates provide useful functions, macros, and types for use in Movement SDK projects. See the [util README](./util/README.md) for more information about the organization of utility crates.
```

Comment:
Let's change the title to "Movement" while we're at it. This will be our main repo going froward and of course that's what the repo is actually called.

Reply:
Okay, 

* title has been changed 
* logo added
* subheader in Troubleshooting section added